pkg/project/project.go

```diff
@@ -7,4 +7,9 @@ const (
 
 const (
 	SystemImageVersionAnn = "field.cattle.io/systemImageVersion"
+	ProjectIDAnn          = "field.cattle.io/projectId"
+)
+
+var (
+	SystemProjectLabel = map[string]string{"authz.management.cattle.io/system-project": "true"}
```

Comment:
Maybe you can create a new commit to change all the SystemProjectLabel variables pointing to the same place.